new/INS1006FCCOS.xml

```diff
@@ -48,8 +48,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <placeName xml:lang="en" xml:id="n1">Franciscan Center of Christian Oriental Studies</placeName>
                     <placeName xml:lang="en" xml:id="n2">Franciscan Center for Christian Oriental Studies</placeName>
                     <placeName xml:lang="fr" xml:id="n3">Centre d'Etudes Orientales Chrétiennes du Mouski</placeName>
-                    <country ref="wd:Q79"/>               
-                    <settlement type="city" ref="wd:Q85"/>
+                    <country ref="wd:Q79">Egypt</country>               
+                    <settlement type="city" ref="wd:Q85">Cairo</settlement>
```

Comment:
this was not what I meant; the name would be printed OK also with wd: link, but if you want to search for all references to a place/person it must be a reference to a Bm ID https://github.com/BetaMasaheft/Documentation/issues/2065